### PR TITLE
Coverage

### LINKED ISSUE #10: add code coverage to CI

### PR DESCRIPTION
**Pull request by czemanb**
Add code coverage to GitHub Actions.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -30,3 +30,13 @@ jobs:
...
       - name: Run behave tests
         run: behave
+        
+      - name: Generate Report
+        run: |
+          pip install coverage
+          coverage run -m unittest
+          
+      - name: Upload Coverage to Codecov
+        uses: codecov/codecov-action@v1
```
Comment (Csaba591) [inline .github/workflows/main.yml:40]: All good! 👌

File: codecov.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Csaba591) [inline codecov.yml:1]: What's the purpose of this file? 🤔

**Review by Csaba591 — CHANGES_REQUESTED** [review]
Good job on the workflow yaml! What is codecov.yml used for?

File: codecov.yml (hunk 2 above)
Comment (czemanb) [reply codecov.yml:1]: One great feature that Codecov offers is the ability to add coverage reports right in your github pull requests. All you need to do is add a codecov.yml file in the root of your project. The file can be empty, it just needs to exist.

**Pull request merged**